scripts/getVestingContracts.js

```diff
@@ -0,0 +1,21 @@
+const GenesisManager = artifacts.require("GenesisManager")
+const genesisConfig = require("../migrations/genesis.config.js")
+
+module.exports = async () => {
+    const genesisManager = await GenesisManager.at("0x9a4d4cafbb1fc3b00d387095144e34b993d82d9a")
```

Comment:
The address passed in here will change from deployment to deployment - I think we could change this to `await GenesisManager.deployed()` which will always work right after deployment (from the machine that the deployment account lives on) and if the script ever needs to be run somewhere else where the Truffle artifacts aren't present there can be hand made change to pass in the address